diff -Naur gst-plugins-bad.orig/tests/check/pipelines/ipcpipeline.c gst-plugins-bad/tests/check/pipelines/ipcpipeline.c
--- gst-plugins-bad.orig/tests/check/pipelines/ipcpipeline.c	2018-03-27 15:52:44.351861610 +0200
+++ gst-plugins-bad/tests/check/pipelines/ipcpipeline.c	2018-03-27 15:54:40.368868300 +0200
@@ -27,7 +27,10 @@
 #endif
 
 #define _GNU_SOURCE             /* See feature_test_macros(7) */
+#ifdef HAVE_PIPE2
+/* Only include glibc's unistd.h if we have pipe2 support in the kernel, else we don't need it */
 #include <unistd.h>
+#endif
 #include <stdio.h>
 #include <fcntl.h>
 #include <sys/wait.h>
